README.md

```diff
@@ -1,16 +1,105 @@
 ![](https://github.com/projectriff/charts/workflows/CI/badge.svg)
 
-# projectriff Helm charts
+# projectriff Helm charts (and uncharts)
 
-Helm charts to install Istio and riff.
+Helm charts (and uncharts) to install Istio and riff.
 
-## Install
+## Install (kapp)
 
 ### Prerequisites
 
 - a running kubernetes cluster (1.14+)
-- kubectl (1.14+)
-- helm (2.13+)
+- [kubectl](https://kubectl.docs.kubernetes.io) (1.14+)
+- [kapp](https://get-kapp.io) (0.14+)
+- [ytt](https://get-ytt.io) (0.14+)
```

Comment:
is this the minimum required version? current is `v0.22.0`

Reply:
minimum'ish. It's what CI is currently using. I use 0.22 locally.